integration-tests/src/test/resources/resize/readonlyrest.yml

```diff
@@ -0,0 +1,18 @@
+readonlyrest:
+  ssl:
+    enable: true
+    keystore_file: "keystore.jks"
+    keystore_pass: readonlyrest
+    key_pass: readonlyrest
+
+  access_control_rules:
+
+    # ES containter initializer need this rule to configure ES instance after startup
+    - name: "CONTAINER ADMIN"
+      verbosity: error
+      type: allow
+      auth_key: admin:container
+
+    - name: "test1"
+      indices: ["test1_index", "test1_index_resized"]
```

Comment:
it would be nice to test indices rule in this configuration:
`indices: ["test1_*"]`